src/Api/Vault/Controllers/CiphersController.cs

```diff
@@ -345,6 +346,87 @@ private async Task<bool> CanEditCipherAsAdminAsync(Guid organizationId, IEnumera
         return await CanEditCiphersAsync(organizationId, cipherIds);
     }
 
+    private async Task<bool> CanDeleteOrRestoreCipherAsAdminAsync(Guid organizationId, IEnumerable<Guid> cipherIds)
```

Comment:
👏🏼 Thanks for all the helpful comments

Reply:
:question: I see a `TODO` does a ticket currently track this?

Reply:
When we were planning this work it was mentioned that all the logic in this controller will be refactored at some point by the Vault team. @shane-melton do you know if any ticket exists for this?